test/generated_inputs/BUILD.bazel

```diff
@@ -28,6 +28,7 @@ rust_library(
         ":src.rs",
     ],
     edition = "2018",
+    tags = ["norustfmt"],
```

Comment:
All the tagging here is to work around the following error when a target has a mix of local and generated source files.
```
Error writing files: failed to resolve mod `src`: /Users/user/Code/rules_rust/test/generated_inputs/src.rs does not exist
Error writing files: failed to resolve mod `src`: /Users/user/Code/rules_rust/test/generated_inputs/src.rs does not exist
Error writing files: failed to resolve mod `generated_in_different_cfg`: /Users/user/Code/rules_rust/test/generated_inputs/generated_in_different_cfg.rs does not exist
```

This error comes from `rustfmt` but given that the functionality for supporting generated sources in `rules_rust` is new, I'd rather try to account for this in a separate PR.